### DISCUSSION TITLE
Enforcing values for parameters for a given `aws` resource type

### DISCUSSION THREAD

**Discussion by PLBRM** [terraform-linters/tflint-ruleset-aws#597] [Q&A] [answered]
Say, in my team's terraform projects, I wanted to ensure that all `aws_dynamodb_table`s that are declared have their `billing_mode` parameter set to `ON_DEMAND`. Is there a rule that I could use in `tflint-ruleset-aws` that would enable me to lint for this? I was having trouble finding related closed issues.

**Comment by wata727** [comment, marked as answer]
Probably a good place to start is with OPA ruleset.
https://github.com/terraform-linters/tflint-ruleset-opa
